xc7/utils/prjxray_form_channels.py

```diff
@@ -139,15 +141,10 @@ def build_tile_type_indicies(c):
     c.execute("CREATE INDEX site_pin_index ON site_pin(name, site_type_pkey);")
     c.execute("CREATE INDEX wire_name_index ON wire_in_tile(name, tile_type_pkey);")
     c.execute("CREATE INDEX wire_site_pin_index ON wire_in_tile(site_pin_pkey);")
-    c.execute("CREATE INDEX tile_type_index ON tile(tile_type_pkey);")
```

Comment:
Why removed these indices?

Reply:
These indices were created for the `tile` table right after grid import. Now the grid is imported to the `phy_tile` table which is not used anywhere else but during the location mapping. Once all entries are copied to the `tile` table those indices that you are referring to are created. See lines 745-747 in this file.

Reply:
Okay